ReactiveTask/Task.swift

```diff
@@ -43,6 +43,14 @@ public struct TaskDescription {
 		self.environment = environment
 		self.standardInput = standardInput
 	}
+	
+	/// A GCD group which to wait completion
+	private static var group = dispatch_group_create()
```

Comment:
I think this should be created within `launchTask`, since `TaskDescription`s can be reused across multiple task invocations.